src/lib/plugins/header.js

```diff
@@ -6,5 +6,5 @@ module.exports.player=function(player,serv,settings)
       footer: JSON.stringify(footer)
     });
 
-  player.playerlistUpdateText(settings["player-list-text"]["header"], settings["player-list-text"]["footer"]);
+  //player.playerlistUpdateText(settings["player-list-text"]["header"], settings["player-list-text"]["footer"]);
```

Comment:
Instead of commenting code, delete it if it's not needed (saving old code is Github's job)